tools/pysa_integration_tests/utils.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from __future__ import annotations
+
+import json
+import logging
+import subprocess
+import sys
+from pathlib import Path
+from typing import final, Sequence
+
+LOG: logging.Logger = logging.getLogger(__name__)
+
+
+@final
+class PyreErrorException(Exception):
+    """
+    Custom Exception to raise when Pyre errors out
+    """
+
```

Comment:
nitpick: remove empty line

Reply:
Huh, black requires one such empty line for passing tests :/